fix: update cymbal bank deploy instructions and tfvar examples (#483)
GoogleCloudPlatform/terraform-google-enterprise-application: examples/cymbal-bank/README.md

#### Add Cymbal Bank envs at App Factory

> 1. Navigate to App Factory repository and add the Cymbal Bank application at `terraform.tfvars` if it was not created:

Reviewer: minimal changes, feel free to expand

```suggestion
1. Navigate to App Factory repository and add the Cymbal Bank application in `terraform.tfvars`, if it was not previously created:
```

    ```hcl
    applications = {
        "cymbal-bank" = {
            "balancereader" = {
                create_infra_project = false
                create_admin_project = true
            }
            "contacts" = {
                create_infra_project = false
                create_admin_project = true
            }
            "frontend" = {
                create_infra_project = false
                create_admin_project = true
            }
            "ledgerwriter" = {
                create_infra_project = true
                create_admin_project = true
            }
            "transactionhistory" = {
                create_infra_project = false
                create_admin_project = true
            }
            "userservice" = {
                create_infra_project = true
                create_admin_project = true
            }
        }
    }
    ```

> 1. Navigate to App Factory repository and update `cloudbuildv2_repository_config` at `terraform.tfvars` with cymbal-bank repositories if it was not updated:

Reviewer: minimal changes, feel free to expand

```suggestion
1. Navigate to App Factory repository and update `cloudbuildv2_repository_config` in `terraform.tfvars` to include the cymbal-bank repositories if it was not previously updated:
```
